### PR TITLE
Support sslcertlocation & sslkeylocation for zabbix proxy

### PR DESCRIPTION
**Pull request by shaunbugler**
#### Pull Request (PR) description
Support these params in zabbix::proxy:
- sslcalocation 
- sslcertlocation 
- sslkeylocation

#### This Pull Request (PR) fixes the following issues
zabbix::proxy has supported sslcalocation, sslcertlocation and sslkeylocation since version 2.4, these params were supported in the zabbix proxy template but never added to the zabbix::proxy class

### CODE REVIEW

File: manifests/proxy.pp [hunk 1]
```diff
@@ -424,6 +430,8 @@
   $tmpdir                                   = $zabbix::params::proxy_tmpdir,
   $allowroot                                = $zabbix::params::proxy_allowroot,
   $include_dir                              = $zabbix::params::proxy_include,
+  $sslcertlocation_dir                      = $zabbix::params::proxy_sslcertlocation,
+  $sslkeylocation_dir                       = $zabbix::params::proxy_sslkeylocation,
```
Comment (root-expert) [inline manifests/proxy.pp:434]: Can you use `Optional[Stdlib::Absolutepath]`  type here?

File: manifests/params.pp [hunk 2]
```diff
@@ -386,6 +386,8 @@
   $proxy_snmptrapperfile                    = '/tmp/zabbix_traps.tmp'
   $proxy_sourceip                           = undef
   $proxy_sshkeylocation                     = undef
+  $proxy_sslcertlocation                    = '/usr/lib/zabbix/ssl/certs'
+  $proxy_sslkeylocation                     = '/usr/lib/zabbix/ssl/keys'
```
Comment (root-expert) [inline manifests/params.pp:390]: I think zabbix doesn't have default value. That should be undef in this case.

**Review by root-expert — CHANGES_REQUESTED** [review]
Thank you for the PR, I think `proxy_sslcalocation` is missing as well. If you could add this to this PR would be awesome!

File: manifests/params.pp (hunk 2 above)
Comment (shaunbugler) [reply manifests/params.pp:390]: you are correct, will update

File: manifests/proxy.pp (hunk 1 above)
Comment (shaunbugler) [reply manifests/proxy.pp:434]: Will update

File: templates/zabbix_proxy.conf.erb [hunk 3]
```diff
@@ -406,20 +406,20 @@ Include=<%= @include_dir %>
 #	Location of SSL client certificates.
 #	This parameter is used only in web monitoring.
 #
-<% if @sslcertlocation %>SSLCertLocation=<%= @sslcertlocation %><% end %>
```
Comment (bastelfreak) [inline templates/zabbix_proxy.conf.erb:409]: mhm where did those variables come from? is that something that needs to be cleaned up?

Comment (shaunbugler) [reply templates/zabbix_proxy.conf.erb:409]: They were added with support for zabbix proxy 3.0, but if you check the merge while the template was updated the params were never added, this is what I am fixing

**Pull request merged**